core/src/edu/asu/c3simulator/screens/EmploymentScreen.java

```diff
@@ -0,0 +1,440 @@
+	{
+		private Employee employee;
+		private VerticalGroup employeePane;
+		
+		public HireEmployeeListener(Employee passedEmployee)
+		{
+			employee = passedEmployee;
+		}
+		
+		@Override
+		public void clicked(InputEvent event, float x, float y)
+		{
+			newEmployeeOptionsPanel.clear();
+			addEmployee(employee, employeePane);
+		}
+	}
+	
+	@SuppressWarnings("unused")
+	private class RemoveEmployeeListener extends ClickListener
+	{
+		//TODO
+	}
+	
+	private static final int DESIGN_WIDTH = 1280;
+	private static final int DESIGN_HEIGHT = 720;
+	private static final int DESIGN_SCREEN_CENTER_X = DESIGN_WIDTH / 2;
+	private static final int DESIGN_SCREEN_CENTER_Y = DESIGN_HEIGHT / 2;
+	private static final int ROSTER_OFFSET_X = 500;
+	private Actor employeePay;
+	private Label employeeName;
+	private Label employeePosition;
+	private Label employeePreferredHourlyWage;
+	private Label employeeMorale;
+	private Label netBonuses;
+	private Label averageAnnualBonus;
+	private Label averageAnnualRaise;
+	private Label netEarnings;
+	private TextField employeePayField;
+	private Table employeePayTable;
+	private VerticalGroup newEmployeeOptionsPanel;
+	
+	
+	@SuppressWarnings("unused")
+	private Game game;
+	private Skin skin;
+	private Stage stage;
+	private Employee selectedEmployee;
+	
+	/**
+	 * Used in order to transition between the Main Hub, Business Management, and
+	 * Employment Screen.
+	 * 
+	 * @param game
+	 */
+	public EmploymentScreen(Game game)
+	{
+		this.game = game;
+		
+		Viewport stageViewport = new StretchViewport(DESIGN_WIDTH, DESIGN_HEIGHT);
+		this.stage = new Stage(stageViewport);
+		this.skin = new Skin(Gdx.files.internal("skins/default/uiskin.json"));
+		
+		Table roster = new Table();
+		
+		Actor employeePane = createEmployeePane();
+		Actor employeeModel = createEmployeeModel();
+		
+		roster.add(employeeModel).top().spaceRight(75);
+		roster.add(employeePane).top().spaceLeft(75);
+		
+		roster.setTransform(true);
+		roster.setPosition(DESIGN_SCREEN_CENTER_X, DESIGN_SCREEN_CENTER_Y);
+		
+		// TODO: Corner Advisor		
+		employeePane.setPosition(DESIGN_WIDTH / 4 + ROSTER_OFFSET_X, DESIGN_HEIGHT / 6);
+		employeePane.setSize(350, 400);
+		roster.setPosition(ROSTER_OFFSET_X, DESIGN_HEIGHT / 2);
+		
+		stage.addActor(roster);
+		stage.addActor(employeePane);
+		
+	}
+	
+	/**
+	 * Company creates a list of employees of the type Employee and
+	 * returns that list.
+	 * 
+	 * @return
+	 */
+	public Company getCompanyContext()
+	{
+		return new Company() {
+			
+			@Override
+			public List<Employee> getEmployees()
+			{
+				Employee employee1 = new Employee("Jason Richards", "Manager", 14, 11,
+						10, 1600, 15.6f, 3.2f, 150);
+				Employee employee2 = new Employee("Janet Wilmore", "Product Creation", 8,
+						8, 10, 1000, 6.7f, 1.2f, 35);
+				
+				List<Employee> employees = new LinkedList<>();
+				employees.add(employee1);
+				employees.add(employee2);
+				
+				return employees;
+			}
+			
+		};
```

Comment:
This could be its own class. Call it CompanyStub and put it in the testing/stubs package